scripts/new_project.kts

```diff
@@ -61,6 +63,27 @@ object NewProject {
         val process = Runtime.getRuntime().exec(command)
         process.inputStream.reader().forEachLine { println(it) }
     }
+
+    private fun renamePackageNameWithinFiles() {
+        showMessage("=> 🔎 Renaming package name within files...")
+        File("../$appNameWithoutSpace")
```

Comment:
also, please rebase #218 to get `projectPath` and update it here 😄

Reply:
@kaungkhantsoe One more change here 🙇

Reply:
@Tuubz @manh-t  Updated 🚀